x-pack/legacy/plugins/graph/public/templates/index.html

```diff
@@ -74,7 +74,7 @@
     </div>
   </div>
 
-  <div class="config" ng-show="kbnTopNav.currentKey === 'fields'" data-test-subj="fieldSelectionList">
+  <div class="config" ng-if="currentlyDisplayedKey === 'fields'" data-test-subj="fieldSelectionList">
```

Comment:
@lizozom  These changes seem to break something about the data binding - try to add a vertex type and type something in the filter bar - this error will show up:
```
angular.js:14800 TypeError: Cannot read property 'toLowerCase' of null
    at ChildScope.$scope.filterFieldsKeyDown (app.js:554)
    at fn (eval at compile (angular.js:NaN), <anonymous>:4:177)
    at ChildScope.$eval (angular.js:18542)
    at eval (angular.js:26439)
    at Object.eval (angular.js:29324)
    at forEach (angular.js:408)
    at Object.$$writeModelToScope (angular.js:29322)
    at writeToModelIfNeeded (angular.js:29315)
    at eval (angular.js:29309)
    at validationDone (angular.js:29234)
```

and it won't select the field you actually selected. When using `ng-show` instead it's working fine for me